packages/portalnetwork/src/wire/utp/Socket/UtpSocket.ts

```diff
@@ -198,6 +198,15 @@ export class UtpSocket extends EventEmitter {
     return this.readerContent
   }
 
+  async handleResetPacket() {
```

Comment:
Will this socket get garbage collected at this point by the `uTpProtocol` class?